solutions/my_zip_solution.rb

```diff
@@ -1,4 +1,12 @@
 class Array
+  # Define a method `my_zip(*arrays)` that merges 
```

Comment:
Should be `Array#my_zip` or otherwise specify that it's monkey patched onto the Array class